vowpalwabbit/io_to_queue.h

```diff
@@ -69,22 +52,34 @@ inline bool read_input_file_ascii(vw& all, char *&line) {
 }
  
 inline bool read_input_file_binary(vw& all, char *&line) {
- 
- size_t total_num_input_bytes = 0;
- 
- //read_all_data returns the number of bytes successfully read from the input files
- total_num_input_bytes = all.example_parser->input->read_all_data(line, total_num_input_bytes);
- 
- bool finish = true;
- 
+
+  size_t example_length_bytes;
+  io_buf* input = all.example_parser->input.get();
+
+  // Return true if no examples found in file.
```

Comment:
True if n examples found seems counterintuitive to me

Reply:
Just modified the comment. Maybe it makes more sense now.